src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -10,7 +10,12 @@ public class AuthenticationService {
      * @return true if user by email exists and passed password is equal to user's password.
      * Return false in any other cases.
      */
+
     public boolean login(String email, String password) {
+        UserService userService = new UserService();
```

Comment:
Is it a good choice to create `new UserService();` on every call of method `login`?